pandas/io/json/_json.py

```diff
@@ -607,6 +615,9 @@ def read_json(
 
         .. versionadded:: 1.3.0
 
+    engine : {{'ujson', 'pyarrow'}}
```

Comment:
This should default to `"ujson"`

Reply:
I actually have a question about this. I thought that the `engine` argument will be optional. Or does the `read_json` method uses ujson currently and we are just adding another option?

Reply:
The existing parsing logic is (vendored) ujson code, so that's why the "default" should be "ujson".

Reply:
Yeah after I made that comment that's what I figured.